app/src/main/java/org/simple/clinic/home/patients/links/PatientsTabLinkModel.kt

```diff
@@ -0,0 +1,24 @@
+package org.simple.clinic.home.patients.links
+
+import android.os.Parcelable
+import kotlinx.parcelize.Parcelize
+import org.simple.clinic.facility.Facility
+
+@Parcelize
+data class PatientsTabLinkModel(
+    val facility: Facility?,
+) : Parcelable {
+  companion object {
+    fun default() = PatientsTabLinkModel(
+        facility = null,
+    )
+  }
+
+  val monthlyScreeningReportsEnabled: Boolean
+    get() = facility?.config?.monthlyScreeningReportsEnabled == true
```

Comment:
Should also check there is a questionnaire of type `monthly_screening_reports` in the DB.

Reply:
@prabhanshuguptagit will add in the separate PR. Thanks for the info.